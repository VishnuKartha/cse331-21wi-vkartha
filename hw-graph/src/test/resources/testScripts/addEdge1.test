# Creates a single graph and adds 2 nodes to it and an edge between the nodes

# Create a Graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 n1
AddNode graph1 n2

# Add an edge between the two nodes
AddEdge graph1 n1 n2 e1