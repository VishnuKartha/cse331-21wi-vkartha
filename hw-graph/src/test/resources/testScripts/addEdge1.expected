# Creates a single graph and adds 2 nodes to it and an edge between the nodes

# Create a Graph
created graph graph1

# Add a pair of nodes
added node n1 to graph1
added node n2 to graph1

# Add an edge between the two nodes
added edge e1 from n1 to n2 in graph1